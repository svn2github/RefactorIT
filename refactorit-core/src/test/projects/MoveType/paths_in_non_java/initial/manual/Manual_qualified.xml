<root>com.test.main.Test is a very good class. Definitely useful.</root>
