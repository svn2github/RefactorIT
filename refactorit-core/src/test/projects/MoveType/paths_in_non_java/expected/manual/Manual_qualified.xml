<root>org.test.simple.Test is a very good class. Definitely useful.</root>
